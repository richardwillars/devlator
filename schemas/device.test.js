const deviceSchema = require("./device");

describe("schemas/device", () => {
  it("should export the schema", () => {
    expect(deviceSchema).toMatchSnapshot();
  });
});
