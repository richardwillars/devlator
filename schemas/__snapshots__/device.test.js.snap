// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`schemas/device should export the schema 1`] = `
Object {
  "_id": Object {
    "required": true,
    "type": [Function],
    "unique": true,
  },
  "additionalInfo": Object {
    "default": Object {},
    "required": false,
    "type": [Function],
  },
  "commands": Object {
    "default": Object {},
    "required": true,
    "type": [Function],
  },
  "created": Object {
    "default": [Function],
    "required": false,
    "type": [Function],
  },
  "driver": Object {
    "required": true,
    "type": [Function],
  },
  "events": Object {
    "default": Object {},
    "required": true,
    "type": [Function],
  },
  "name": Object {
    "required": true,
    "type": [Function],
  },
  "originalId": Object {
    "required": true,
    "type": [Function],
  },
  "type": Object {
    "required": true,
    "type": [Function],
  },
}
`;
